.travis.yml

```diff
@@ -1,13 +1,17 @@
+sudo: required
```

Comment:
This is the default for repositories created before 2015. I am just making it explicit.
This is required for using Ubuntu 14.04 (Trusty).